src/sentences/utils.rs

```diff
@@ -200,6 +200,79 @@ mod tests {
         assert_eq!(time.nanosecond(), 500_000_000);
     }
 
+    #[test]
+    #[should_panic]
```

Comment:
Which of the 2 tests here should panic? Maybe you can split them if they both need to panic as this will only check the first one